events_ingress/src/main/java/com/adobe/event/publish/model/CloudEvent.java

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright 2017 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+package com.adobe.event.publish.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.JsonNode;
+import java.util.Objects;
+import java.util.UUID;
+import org.apache.commons.lang3.StringUtils;
+
+public class CloudEvent {
```

Comment:
We can use the Cloud event java SDK instead. https://github.com/cloudevents/sdk-java

Reply:
yes I guess we could ... but that would require quite a bit of work
let me add this to the project backlog for now: https://github.com/adobe/aio-lib-java/issues/16